source/common/protobuf/utility.cc

```diff
@@ -31,7 +31,11 @@ void MessageUtil::loadFromFile(const std::string& path, Protobuf::Message& messa
 }
 
 Json::ObjectSharedPtr WktUtil::getJsonObjectFromStruct(const Protobuf::Struct& message) {
-  Protobuf::util::JsonOptions json_options;
+  Protobuf::util::JsonPrintOptions json_options;
+  // By default, proto field names are converted to camelCase when the message is converted to JSON.
+  // Setting this option makes debugging easier because it keeps field names consistent in JSON
+  // printouts.
+  json_options.preserve_proto_field_names = true;
```

Comment:
Sorry I can't keep track of which is our code and which is proto code. As long as we have some wrapper sounds great.